src/templates/anchor-basic-content.twig

```diff
@@ -12,7 +12,7 @@
   {% embed 'partials/sidebar-layout.twig' %}
     {% block nav %}
 
-      <div class="mb-8">
+      <div class="mb-4 mb-md-6">
```

Comment:
Should this class change be made _only_ on the anchor site, or can it be applied to basic content on the schools sites as well?

Reply:
School as well